crates/router/src/core/errors/api_error_response.rs

```diff
@@ -49,6 +49,8 @@ pub enum ApiErrorResponse {
     InvalidDataValue { field_name: &'static str },
     #[error(error_type = ErrorType::InvalidRequestError, code = "IR_08", message = "Client secret was not provided")]
```

Comment:
Please use consistent terminology in error messages. (i.e) "The client_secret was not provieded" instead of "Client secret..."

Reply:
This error message will not be sent in the api response, this is for internal use only. Have changed it anyways.